frontend/gpay_agent/android/app/src/main/AndroidManifest.xml

```diff
@@ -50,4 +50,6 @@
             android:name="flutterEmbedding"
             android:value="2" />
     </application>
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
We only need to request one access permission
https://developers.google.com/maps/documentation/android-sdk/location#location_permissions

Reply:
Removed ACCESS_COARSE_LOCATION